.github/workflows/build.yml

```diff
@@ -27,28 +28,22 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Use OCaml ${{ matrix.ocaml-version }}
-        uses: avsm/setup-ocaml@v2
+      - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        uses: ocaml/setup-ocaml@v2
         with:
-          ocaml-version: ${{ matrix.ocaml-version }}
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          opam-local-packages: libdash/*.opam
```

Comment:
This ensures that the setup-ocaml auto-pin and auto-depext functionalities works properly in the directory.
In fact, as you can see in this log, it installed the dev version correctly.
https://github.com/smorimoto/smoosh/runs/5330079707?check_suite_focus=true#step:4:57

Reply:
🤯